app/components/samples/table_component.en.yml

```diff
@@ -9,4 +9,4 @@ en:
   counts:
     samples: Samples
     selected: Selected
-  select_all: Select All
+  select_page: Select / Deselect Samples
```

Comment:
**nitpick (blocking):** Can you change this translation to `Select / Deselect visible sample` in both translation files please.

Reply:
0d1b2486258a47eb15a591b32664b27022d1ca2d